package.json

```
@@ -28,6 +29,7 @@
         "embla-carousel-autoplay": "8.6.0",
         "embla-carousel-react": "8.6.0",
         "input-otp": "1.4.2",
+        "jsonwebtoken": "^9.0.2",
```

Biome (2.1.2)

[error] The key jsonwebtoken was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)